src/adapters/source-adapters/UniswapAnchoredViewSourceAdapter.sol

```diff
@@ -63,6 +62,16 @@ abstract contract UniswapAnchoredViewSourceAdapter is SnapshotSource {
         return (DecimalLib.convertDecimals(sourcePrice, SOURCE_DECIMALS, 18), latestTimestamp);
     }
 
+    /**
+     * @notice Returns the requested round data from the source.
+     * @dev UniswapAnchoredView does not support this and returns uninitialized data.
+     * @return answer Round answer in 18 decimals.
+     * @return updatedAt The timestamp of the answer.
+     */
+    function getSourceDataAtRound(uint256 /* roundId */ ) public view virtual override returns (int256, uint256) {
```

Comment:
does it make sense to return `(0,0)` or revert if the adapter does not support round IDs? perhaps `(0,0)` at source and revert at destination?

Reply:
not sure if its worth reverting within Oval. clients that request `getRoundData` by roundId anyway should be able to handle uninitialized values as Chainlink can return `(0, 0)` if the data is not available for requested round.

Reply:
ok so, related the other thread, we design for never reverting for users, even if it means returning incomplete data.

Reply:
yes, I think we better avoid revers